lib/memory.js

```diff
@@ -49,6 +50,7 @@ module.exports.startMemoryMeasurement = function (test, options, time, graphData
         options = options || {};
 
         _.defaults(options, {
+            dir: './docs/memory',
```

Comment:
This is a bit invasive, and breaks backwards-compatibility. Could you please change it to use the current directory as the default, so that the behavior does not change?